### ISSUE TITLE
'DatabricksSQLConnectionWrapper' object has no attribute 'cancel'

### ISSUE THREAD

**Issue by xg1990** [databricks/dbt-databricks#184]
### Describe the bug

when running `dbt build`, got the following error msg

```
06:26:51  Encountered an error:
'DatabricksSQLConnectionWrapper' object has no attribute 'cancel'
06:26:51  Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/dbt/task/runnable.py", line 379, in execute_nodes
    self.run_queue(pool)
  File "/usr/local/lib/python3.9/site-packages/dbt/task/runnable.py", line 293, in run_queue
    self._raise_set_error()
  File "/usr/local/lib/python3.9/site-packages/dbt/task/runnable.py", line 274, in _raise_set_error
    raise self._raise_next_tick
dbt.exceptions.FailFastException: FailFast Error in model ******(models/******.sql)
  Failing early due to test failure or runtime error
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/dbt/main.py", line 129, in main
    results, succeeded = handle_and_check(args)
  File "/usr/local/lib/python3.9/site-packages/dbt/main.py", line 191, in handle_and_check
    task, res = run_from_args(parsed)
  File "/usr/local/lib/python3.9/site-packages/dbt/main.py", line 238, in run_from_args
    results = task.run()
  File "/usr/local/lib/python3.9/site-packages/dbt/task/runnable.py", line 470, in run
    result = self.execute_with_hooks(selected_uids)
  File "/usr/local/lib/python3.9/site-packages/dbt/task/runnable.py", line 433, in execute_with_hooks
    res = self.execute_nodes()
  File "/usr/local/lib/python3.9/site-packages/dbt/task/runnable.py", line 382, in execute_nodes
    self._cancel_connections(pool)
  File "/usr/local/lib/python3.9/site-packages/dbt/task/runnable.py", line 357, in _cancel_connections
    for conn_name in adapter.cancel_open_connections():
  File "/usr/local/lib/python3.9/site-packages/dbt/adapters/base/impl.py", line 1032, in cancel_open_connections
    return self.connections.cancel_open()
  File "/usr/local/lib/python3.9/site-packages/dbt/adapters/sql/connections.py", line 43, in cancel_open
    self.cancel(connection)
  File "/usr/local/lib/python3.9/site-packages/dbt/adapters/spark/connections.py", line 296, in cancel
    connection.handle.cancel()
AttributeError: 'DatabricksSQLConnectionWrapper' object has no attribute 'cancel'

```

### Steps To Reproduce

using the following version:

dbt-databricks==1.2.2
dbt-core==1.2.1

### Expected behavior
No error is expected.

### Screenshots and log output
as provided above

### System information

dbt-databricks==1.2.2
dbt-core==1.2.1

**The output of `python --version`:**

### Additional context

Might be related to this PR: https://github.com/databricks/dbt-databricks/pull/163/files

**Comment by ueshin** [comment]
Hi @xg1990, thanks for the report!

It's a know bug and already fixed at #173. We will publish the next maintenance release any time soon.
We have published the RC of the release: [1.2.3rc0](https://pypi.org/project/dbt-databricks/1.2.3rc0/) and [1.1.5rc0](https://pypi.org/project/dbt-databricks/1.1.5rc0/).

Thanks.

**Comment by xg1990** [comment]
Thank you! Really appreciate your help!

**Comment by ueshin** [comment]
Closing as we published [1.2.3](https://pypi.org/project/dbt-databricks/1.2.3/) and [1.1.5](https://pypi.org/project/dbt-databricks/1.1.5/) that include the fix.
